lms/static/scripts/frontend_apps/components/EmailPreferencesApp.tsx

```diff
@@ -10,17 +12,49 @@ export default function EmailPreferencesApp() {
   const onSave = useCallback(() => setSaving(true), []);
 
   return (
-    <div className="h-full grid place-items-center">
-      <div className="w-96">
-        <EmailPreferences
-          selectedDays={selectedDays}
-          updateSelectedDays={newSelectedDays =>
-            setSelectedDays(prev => ({ ...prev, ...newSelectedDays }))
-          }
-          onSave={onSave}
-          saving={saving}
+    <div className="min-h-full bg-grey-2">
+      <div
+        className={classnames(
+          'flex justify-center p-3 w-full',
+          'bg-white border-b shadow'
+        )}
+      >
+        <img
+          alt="Hypothes.is logo"
+          src="/static/images/email_header.png"
+          className="h-10"
         />
       </div>
+      <div className="flex flex-col gap-y-4 py-8">
+        <div className="max-w-[450px] mx-auto">
+          <EmailPreferences
+            selectedDays={selectedDays}
+            updateSelectedDays={newSelectedDays =>
+              setSelectedDays(prev => ({ ...prev, ...newSelectedDays }))
+            }
+            onSave={onSave}
+            saving={saving}
+          />
+        </div>
+        <p className="text-center text-grey-8">
+          <InfoIcon className="inline" /> Do you need help? Check our{' '}
+          <Link
+            variant="custom"
+            underline="always"
+            href="https://web.hypothes.is/help/"
+          >
+            help topics
+          </Link>{' '}
+          or{' '}
+          <Link
+            variant="custom"
+            underline="always"
+            href="https://web.hypothes.is/get-help/"
+          >
+            create a ticket
+          </Link>
```

Comment:
I would prefer to have just one link here, since https://web.hypothes.is/help/ has a prominent link to create a ticket on it anyway. Maybe something like "Do you need help? Visit our [help center](https://web.hypothes.is/help/)." That way we can more easily keep it up to date if the support team decide to recommend other flows in future.